Model/src/GamePackage/Bank.java

```diff
@@ -1,7 +1,158 @@
 package GamePackage;
 
-/**
- * Created by Dylan Kernohan on 2/10/18.
- */
+import BoardPackage.BoardSquare;
+import BoardPackage.PropertySquare;
+
+import java.util.ArrayList;
+
+/****************************************************************************************************************
+ * The bank class is responsible of keeping track of the game's houses, player montages, and also perform
+ * certain transactions.
+ *
+ * @author Santiago Quiroga
+ * @version 2/13/2018
+ ***************************************************************************************************************/
 public class Bank {
+    /* Number of houses and Hotels */
+    private int numHouses, numHotels;
+
+    /* List of properties owned and properties mortgaged */
+
+    private ArrayList<PropertySquare> propertiesOwned, propertiesMortgaged;

```

Comment:
This should be an ArrayList of all ownable properties. Perhaps we should make a new type of BoardSquare class called OwnableSquare and then have all of the ownable board squares(RR, Properties, and Utilities) extend that class. This would allow all of the ownable boardSquares in that ArrayList.

Reply:
I agree. the only thing is that we would have to make an interface, since we're already extending the BoardSquare class. Also this would affect the way that the player class is written.

Reply:
In oder words, we need a sepparate issue to write and implement the ownableSquare class.

Reply:
If we make this an interface, we can move a lot of duplicate variables. I like this idea. Will need to change a few things in player though.

Reply:
add this to an issue for the OwnableSquare